src/views/iframe.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <iframe id="iframe" :src="`${baseURL}/?iframe`" scrolling="auto"></iframe>
```

Comment:
```suggestion
  <iframe id="iframe" :src="/" scrolling="auto"></iframe>
```

Reply:
Le code actuel ne fonctionne pas sur la deploy-preview

Reply:
Je n'ai pas réussi à  bien configurer les variables d'environnement sur netlify pour que `MES_AIDES_ROOT_URL` soit égal à [`DEPLOY_URL`](https://docs.netlify.com/configure-builds/environment-variables/#deploy-urls-and-metadata).

Reply:
Je viens de proposer un nouveau changement / en /?iframe